### ISSUE TITLE
Need friendly intellisense descriptions for procedures in the standard library

### ISSUE THREAD

**Issue by WaynePhillipsEA** [WaynePhillipsEA/twinbasic#26]
**Is your feature request related to a problem? Please describe.**
At the moment, we've only got detailed intellisense descriptions for MsgBox and InputBox [as of 0.9.1602].  

**Describe the solution you'd like**
All standard library functions, and enumerations etc should have nice detailed descriptions in intellisense / signature help.

**Comment by retailcoder** [comment]
If it doesn't entail much additional work, could the docstrings be open-sourced in some way? Rubberduck would definitely load them and display them if they were available somewhere!

**Comment by WaynePhillipsEA** [comment]
Yes, absolutely.  Really good idea.  A good format for twinBASIC would be a JSON based format, with the text being markdown.  We'd need to consider language awareness -- perhaps a single JSON file per language/locale.  

Currently, twinBASIC uses the full namespace IDs to match up the markdown text to the corresponding library procedure;  e.g. 'VBA.Interaction.InputBox'.  The idea was that later we could take it a step further and have this help text available for other third-party libraries like ADODB...

We could do it here, or perhaps in a separate repository, with a very permissive licence?
